### DISCUSSION TITLE
Problem when using transforms.FiveCrop()

### DISCUSSION THREAD

**Discussion by MananShukla7** [mrdbourke/pytorch-deep-learning#378] [Q&A]
Hello,
I encountered  this problem when i was trying out FiveCrop augmentation technique and tried to plot the transformed image.

The trasformation i was using was-

![image](https://user-images.githubusercontent.com/68244767/228884207-ef8c2553-ffdb-4250-a831-7ec77f951ead.png)

and the error i got is:
![image](https://user-images.githubusercontent.com/68244767/228884663-8e3ce5e3-9295-4f32-acc0-9f3d63713abd.png)

Upon searching i figured out that it was due to fact that the FiveCrop tranformation returns a tuple of image array and that we have to use Lambda to resolve this issue.
I am not able to fully understand why are we using Lambda and how is it solving the issue
Can somebody explain it to me in a simpler way??

I found out about lamda from this:https://stackoverflow.com/questions/66837824/pic-should-be-pil-image-or-ndarray-got-class-tuple

**Comment by sorker** [comment]
i also get this problem, do you resloved?

**Reply by dmahdipour** [reply]
I've got this too.

**Reply by MananShukla7** [reply]
You can check the stack overflow link for solution.
